### PR TITLE
Wrong 18next english cookie value fix

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Fixes wrong i18next cookie value for english being set

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/resources/LoadLocalizations/LoadLocalizations.tsx [hunk 1]
```diff
@@ -9,7 +9,18 @@ interface Props {
 }
 
 const LoadLocalizations = ({ children }: Props) => {
-  const backupLang = getCookie('i18next') ? getCookie('i18next') : 'no_nb';
+  const getBackupLanguage = () => {
+    const value = getCookie('i18next');
+    if (value === 'no_nn') {
```
Comment (acn-dgopa) [inline src/resources/LoadLocalizations/LoadLocalizations.tsx:14]: Just trying to understand this code block. Are you trying to return a default language? Doesn't BFF already defaults language to bokmål?

**Review by allinox — APPROVED** [review]
Good catch!

File: src/resources/LoadLocalizations/LoadLocalizations.tsx (hunk 1 above)
Comment (Albertlarsen) [reply src/resources/LoadLocalizations/LoadLocalizations.tsx:14]: It's meant as a second verification for i18next cookie corresponding with actual localization file names

Comment (acn-dgopa) [reply src/resources/LoadLocalizations/LoadLocalizations.tsx:14]: But you are returning the same text as the text in "if" condition.

Comment (Albertlarsen) [reply src/resources/LoadLocalizations/LoadLocalizations.tsx]: Yeah that was redundant, changed it to just return value now

**Pull request merged**